obconf-qt-git/PKGBUILD

```diff
@@ -0,0 +1,36 @@
+# Maintainer: Jerome Leclanche <jerome@leclan.ch>
+
+_pkgname=obconf-qt
+pkgname=$_pkgname-git
```

Comment:
This one is okay but lets not use directly from git since they are publishing releases we need to use the release tarballs.